fix-google-analytics/index.html

```diff
@@ -0,0 +1,38 @@
+---
+title: Bliv GDPR-compliant med dine analytics – Fix Google Analytics – deranged
+layout: default
+---
+<h1>Bliv GDPR-compliant med dine analytics</h1>
+
+Google Analytics kan "ikke kan benyttes lovligt uden videre," skriver Datatilsynet i en [nylig konklusion om værktøjet](https://www.datatilsynet.dk/presse-og-nyheder/nyhedsarkiv/2022/sep/brug-af-google-analytics-til-webstatistik). Det gælder altså alle der opererer i Danmark: I skal finde en vej udenom. Så hvad gør man nu?
+
+Heldigvis behvøer man ikke skifte ud af Google Analytics lige med det samme - man kan nemlig sørge for at pille den personhenførbare data ud af det der sendes til Google før de modtager det, og så må man godt bruge det.

```

Comment:
Måske henvise til det fransk datatilsyns beskrivelse? https://www.cnil.fr/en/google-analytics-and-data-transfers-how-make-your-analytics-tool-compliant-gdpr